emulator/src/emu_cnfs.c

```diff
@@ -97,50 +95,34 @@ bool deinitCnfs(void)
     return true;
 }
 
-const uint8_t* cnfsGetFile(const char* fname, size_t* flen)
+const uint8_t* cnfsGetFile(cnfsFileIdx_t fIdx, size_t* flen)
 {
-    if (cnfsInjectedFilename && !strcmp(cnfsInjectedFilename, fname))
+    if (cnfsInjectedFilename && fIdx >= CNFS_NUM_FILES)
```

Comment:
@dylwhich this may need some thinking too

Reply:
Hmm, okay, yeah I'm not sure how to do this now that I take a closer look at it... The whole reason the "injected file" thing exists was for opening MIDI files on-the-fly and letting the player load them... by name. I guess since this really is an emulator-only thing I could just switch to injecting the file data into NVS instead, which works fine, and drop the asset injection feature altogether.